poetry.toml

```diff
@@ -0,0 +1,3 @@
+cache-dir = ".cache"
```

Comment:
@prusnak how important is this?
we're running into https://github.com/python-poetry/poetry/issues/3049 because of it, it's rather unpleasant when doing local development.
would it be OK to remove it until the issue is fixed?

Reply:
I don't mind - I just copied the original setting from `PIPENV_CACHE_DIR`, but have no idea why it was there.

Reply:
I think this was there because of the CI, where there is no `$HOME` and cache defaults to `$HOME/.cache` essentially leading to `/.cache` which is not writable.